packages/sources/layer2-sequencer-health/test/integration/fixtures.ts

```diff
@@ -106,6 +120,20 @@ export const mockResponseFailureHealth = (): void => {
       'Vary',
       'Origin',
     ])
+
+  nock('https://venus.scroll.io/v1/sequencer/status')
+    .get('')
+    .query(() => true)
+    .reply(200, () => ({ errcode: 1, errmsg: 'Mock Error Message', data: { health: 0 } }), [
```

Comment:
I think this is incorrect.
The API returns 1 and 2 for healthy and nonhealthy respectively.
We probably also need to change the handling.